### DISCUSSION TITLE
docker run command outputs on stderr when successful

### DISCUSSION THREAD

**Discussion by tuxmeister** [Jacques44/node-red-contrib-bigssh#32] [Q&A]
Hi there!

I'm wondering if I'm being especially daft but I'm trying using your bigSSH node (wonderful work) to update some containers I run on a regular basis via the watchtower container and a simple "docker run" command:

`docker run --rm -v /var/run/docker.sock:/var/run/docker.sock containrrr/watchtower:latest --no-startup-message --label-enable --run-once --cleanup --debug`

This works well on the system and the output flows great, but through the stderr pipe. When I tried both the example you provided and my own with a simpler command such as docker ps or really anything else, it flows through stdout as expected. 

Is this known behaviour? Am I missing something perhaps about escaping some of the characters? I have also tried with "Payload is an argument" ON and OFF, with the same result. 

Any suggestions?

**Comment by Jacques44** [comment]
Hello. the node seems ok as you can see below.

I've built a flow with 2 commands, one writing to stdout and another one writing to stderr

the node is sending stdout to the first pin and stderr to the 3rd as expected

Is that wiring you made?

![Screen Shot 2021-08-03 at 16 47 14](https://user-images.githubusercontent.com/18165555/128036495-7b278864-43a7-4cfd-a72a-e6c29fdd1bbb.png)
![Screen Shot 2021-08-03 at 16 50 30](https://user-images.githubusercontent.com/18165555/128036717-ffad604b-65bc-4b02-ae9c-29dd29100234.png)
